index.html

```diff
@@ -152,9 +193,9 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
                         </div>
                     </div>
                 </div>
-            </div>
+            </div> -->
```

Comment:
Kindly remove all commented lines of code from your project so as to have a clean code that is easily maintainable 😄